pipeline_dp/dp_engine.py

```diff
@@ -165,6 +165,7 @@ def _select_private_partitions(self, col, max_partitions_contributed: int):
         """
         budget = self._budget_accountant.request_budget(
             mechanism_type=MechanismType.GENERIC)
+        self._budget_accountant.compute_budgets()
```

Comment:
`compute_budgets()` are needed only after finishing constructing of the DP pipeline, namely after the last call of `.aggregate()`, please remove this line

Reply:
👍 Fixed, thank you!